apps/prairielearn/src/question-servers/calculation-subprocess.ts

```diff
@@ -110,52 +115,46 @@ async function callFunction(func, question_course, question, inputData) {
     });
   } catch (err) {
     err.fatal = true;
-    return { data: {}, courseIssues: [err] };
+    return {
+      // We don't have any useful data to return. We'll just lie to the type checker.
```

Comment:
👀 an alternative to this would be to have `callFunction` take in a "default data" object that will be passed back if errors are encountered.